### PR TITLE
Rear restore in ha

### PR DESCRIPTION
**Pull request by hadeskun**
Relax and Recover Scenario fails on ppc64 but not on x86_64, so it is implemented a mechanism to fix on ppc64.

- Related ticket: https://jira.suse.com/browse/TEAM-9655
- Verification run:
> - Build 32.2
http://openqa.suse.de/tests/15867526#
http://openqa.suse.de/tests/15867529#
http://openqa.suse.de/tests/15867599#
http://openqa.suse.de/tests/15867600#
> - Build 36.1
https://openqa.suse.de/tests/15866706#
https://openqa.suse.de/tests/15866708#
https://openqa.suse.de/tests/15866709#
https://openqa.suse.de/tests/15866710#

### CODE REVIEW

File: tests/ha/rear_restore.pm [hunk 1]
```diff
@@ -23,7 +24,18 @@ sub run {
     assert_screen('rear-boot-screen');
     send_key_until_needlematch('rear-recover-selected', 'up');
     send_key 'ret';
-    $self->wait_boot_past_bootloader;
+    # Handle the output error for tty0 in ppc64, more details here TEAM-9655
+    if (is_ppc64le) {
+        wait_screen_change { sleep 10 };
```
Comment (lpalovsky) [inline tests/ha/rear_restore.pm:29]: Is this line necessary here? 
If I understand correctly it just waits 10s after pressing enter in bootloader and screen changes. 
Wouldn't `check_screen` timeout do the job itself? 
I am just thinking how to remove arbitrary sleep call.

File: tests/ha/rear_restore.pm [hunk 2]
```diff
@@ -23,7 +24,18 @@ sub run {
     assert_screen('rear-boot-screen');
     send_key_until_needlematch('rear-recover-selected', 'up');
     send_key 'ret';
-    $self->wait_boot_past_bootloader;
+    # Handle the output error for tty0 in ppc64, more details here TEAM-9655
```
Comment (alvarocarvajald) [inline tests/ha/rear_restore.pm:27]: Not everybody has access to our ticketing system. Best to not reference it in the code. More details about why a simple `wait_boot_past_bootloader` is not enough on ppc64le would be nice to add here as well.

**Review by BillAnastasiadis — APPROVED** [review]
LGTM - please squash commits before or on merge

**Review by alvarocarvajald — COMMENTED** [review]
Can you squash the commits into one? At least this time you appear as the author :)

File: tests/ha/rear_restore.pm (hunk 1 above)
Comment (alvarocarvajald) [reply tests/ha/rear_restore.pm:29]: From https://open.qa/api/testapi/#_wait_screen_change, the code to `wait_screen_change` should be used to cause a change in the screen. `sleep 10` will never do that.

I tend to agree with @lpalovsky that this line is not necessary and the `check_screen` that follows would be enough.

Comment (lpalovsky) [reply tests/ha/rear_restore.pm:29]: Ah ok.. I misunderstood that one. 
Still not necessary, but this one could work right?  `wait_screen_change(send_key 'ret';)`

**Review by mpagot — APPROVED** [review]
LGTM

**Comment by alvarocarvajald** [comment]
Ran the VRs again:

- https://openqa.suse.de/tests/15869603
- https://openqa.suse.de/tests/15869604
- https://openqa.suse.de/tests/15869605
- https://openqa.suse.de/tests/15869606
- https://openqa.suse.de/tests/15869607
- https://openqa.suse.de/tests/15869608
- https://openqa.suse.de/tests/15869609
- https://openqa.suse.de/tests/15869610

**Comment by alvarocarvajald** [comment]
All VRs are green across 2 builds and both architectures. Merging this.

**Comment by lpalovsky** [comment]
LGTM

**Pull request merged**